src/scss/skin-modern/_skin-tv.scss

```diff
@@ -16,7 +16,8 @@
     }
   }
 
-  .#{$prefix}-ui-titlebar-top {
+  .#{$prefix}-ui-titlebar-top,
+  .#{$prefix}-ui-menucaption {
```

Comment:
I guess we don't need this anymore, right?
```suggestion
  .#{$prefix}-ui-titlebar-top {
```

Reply:
Good catch, done in 5efe823e